### ISSUE TITLE
docker push quiet option

### ISSUE THREAD

**Issue by hostingnuggets** [moby/moby#37417]
This is really a feature request suggesting that it would be nice to have a --quiet/-q option to the docker push command.

**Comment by hostingnuggets** [comment]
As a suggestion this -q option might simply display one single line at the end when it's finished such as:

```ea134fe: digest: sha256:5d93220efda2f18783b0584e66d4ac64a53dc6a6dd16aff121b244e5f5326a96 size: 2832```

instead of the whole full progress/output of each steps.

**Comment by thaJeztah** [comment]
Thanks for opening this feature request; this looks to be a duplicate of https://github.com/moby/moby/issues/36655, which was moved to https://github.com/docker/cli/issues/958 because this is something to implement in the client / cli.

Also look at https://github.com/moby/moby/issues/36655#issuecomment-375136087 for a workaround

There's currently a work-in-progress pull request to implement `--quiet` for `docker pull`, but this can likely be implemented similarly for `docker push` once that PR is merged (see https://github.com/moby/moby/issues/13588)

Let me close this in favor of https://github.com/docker/cli/issues/958

**Comment by hostingnuggets** [comment]
Cheers! That's fine by me.
